java-checks-test-sources/src/main/java/checks/LeftCurlyBraceEndLineCheck.java

```diff
@@ -12,10 +16,12 @@
   }
   
   public enum MyEnum {
-    A
+    A(1)
+    // Duplicated issue...
+    // Noncompliant@+1
```

Comment:
What is the same issue reported twice on an enum?

Reply:
The issue was in fact already reported twice before, but the code was not compiling and therefore the strange behavior was not visible.
In fact, this issue was there for many years since the rule is not in Sonarway and the related code is not that frequent, it was unnoticed. Still, it might deserve a ticket to fix this, but with low priority.